fbw-a32nx/src/systems/fmgc/src/guidance/vnav/Predictions.ts

```diff
@@ -87,7 +90,7 @@ export class Predictions {
         const theta2 = Common.getTheta2(theta, mach);
         const delta2 = Common.getDelta2(delta, mach);
         const correctedN1 = EngineModel.getCorrectedN1(commandedN1, theta2);
-        const correctedThrust = EngineModel.tableInterpolation(EngineModel.table1506, correctedN1, mach) * 2 * EngineModel.maxThrust;
+        const correctedThrust = EngineModel.tableInterpolation(EngineModel.table1506, correctedN1, mach) * 2 * config.engineModelParameters.maxThrust;
```

Comment:
What do you intend to do with the tables in `EngineModel`? They should also be aircraft-specific.